ecs-load-old-data/ecs_load_to_s3.py

```diff
@@ -0,0 +1,101 @@
+"""
+Script that gets and removes all data from an RDS, uploading it
+to a CSV file in an S3 bucket in the process.
+"""
+
+from io import StringIO
+from os import environ, _Environ
+
+import pandas as pd
+from boto3 import client
+from mypy_boto3_s3 import S3Client
+from dotenv import load_dotenv
+from sqlalchemy import create_engine, sql, Connection
+
+ARCHIVE_BUCKET = "c9-ladybird-lnhm-data-bucket"
```

Comment:
This is really nice, may be worth changing in some of the files I've worked on too to make it uniform :)